roles/users/README.md

```diff
@@ -0,0 +1,149 @@
+# users
+
+## This role can be used to create users, user groups, LDAP-Auth and to link external and internal user groups
+
+### You can disable certain tasks
+
+#### Tasks to switch on/off
+
++ create_users
++ create_usergroups
++ enable_ldap_auth
++ map_external_usergroups
+
+### Role Variables
+
+#### Example values can be found in defaults/main.yaml
+
+Default:
+
+``` yaml
+server_url: string(url) **required**
+username: string **required**
+password: string
+password: string
+validate_certs: bool defualt (true)
+```
+
+To switch on/off tasks use:
+
+``` yaml
+create_users: bool defualt (true)
```

Comment:
All top level vars need proper namespacing.
...even if it sounds silly.

Reply:
Ok, i´ve changed them.